percona/controllers/pgc/pgc.go

```diff
@@ -447,18 +470,28 @@ func (c *Controller) WorkerCount() int {
 	return c.PerconaPGClusterWorkerCount
 }
 
-func createOrUpdatePrimaryService(clientset kubeapi.Interface, cluster *crv1.PerconaPGCluster) error {
+func createOrUpdateService(clientset kubeapi.Interface, cluster *crv1.PerconaPGCluster, svcType ServiceType) error {
 	ctx := context.TODO()
-	service, err := getPrimaryServiceObject(cluster)
-	if err != nil {
-		return errors.Wrap(err, "get primary service object")
+	var service *corev1.Service
+	switch svcType {
+	case PGPrimaryServiceType:
+		svc, err := getPrimaryServiceObject(cluster)
+		if err != nil {
+			return errors.Wrap(err, "get primary service object")
+		}
+		service = svc
+	case PGBouncerServiceType:
+		svc, err := getPGBouncerServiceObject(cluster)
+		if err != nil {
+			return errors.Wrap(err, "get pgBouncer service object")
+		}
+		service = svc
 	}
-
-	oldSvc, err := clientset.CoreV1().Services(cluster.Namespace).Get(ctx, cluster.Name, metav1.GetOptions{})
+	oldSvc, err := clientset.CoreV1().Services(cluster.Namespace).Get(ctx, service.Name, metav1.GetOptions{})
 	if err != nil {
 		_, err = clientset.CoreV1().Services(cluster.Namespace).Create(ctx, service, metav1.CreateOptions{})
 		if err != nil {
-			return errors.Wrap(err, "create primary service")
+			return errors.Wrapf(err, "create service %s", svcType)
```

Comment:
it's worth to print svc name instead of it's type

Reply:
https://github.com/percona/percona-postgresql-operator/pull/71/commits/a8cbb68cc0989a1d752f02faecd671470266675a